functions/lib/backup.js

```diff
@@ -18,9 +18,12 @@ function Backup (bucketName) {
       name: databaseName,
       outputUriPrefix: bucket,
       collectionIds: [
-        'whitelists',
+        'addendums',
         'agreements',
-        'addendums'
+        'appUsers',
+        'contributions',
+        'events',
+        'whitelists'
```

Comment:
I know this one was not added by this change, but there should be no need to backup whitelists. This collection is generated/updated automatically based on agreement and addendum data. We do update that after every addendum event and every 15 minutes (or so) to be robust against one-off errors.

It is also true that the size here is negligible wrt to storage pricing, so we might as well leave it...